src/main/java/org/gitlab4j/api/models/SystemHook.java

```diff
@@ -64,4 +65,27 @@ public Boolean getEnableSslVerification() {
     public void setEnableSslVerification(Boolean enableSslVerification) {
         this.enableSslVerification = enableSslVerification;
     }
+
+    public void setRepositoryUpdateEvents(Boolean repositoryUpdateEvents) {
+        this.repositoryUpdateEvents = repositoryUpdateEvents;
+    }
+
+    public Boolean getRepositoryUpdateEvents() {
+        return repositoryUpdateEvents;
+    }
+
+    @Override
+    public String toString() {
+        final StringBuilder sb = new StringBuilder("SystemHook{");
```

Comment:
Prefer not to override ```toString()``` like this, these often become out of sync.  An option here would be to implement```toString()``` using something like this:
```
@Override
public String toString() {
    return (new org.gitlab4j.api.utils.JacksonJson().marshal(this));
}
```

This way the output is always in-sync with the class and outputs strings in the expected format, specifically the dates.

We could also consider adding a singleton JacksonJson instance to use for this and elsewhere.  For example, the SystemHookManager class has it's own private instance of JacksonJson, which could use the singleton instance also.